mongodbatlas/resource_mongodbatlas_advanced_cluster.go

```diff
@@ -842,6 +842,9 @@ func expandRegionConfigAutoScaling(tfList []interface{}) *matlas.AdvancedAutoSca
 		apiObject3.MaxInstanceSize = v.(string)
 	}
 
+	apiObject.DiskGB = apiObject2
+	apiObject.Compute = apiObject3
+
```

Comment:
Could we update `resource_mongodbatlas_advanced_cluster_test.go` to include the autoscaling attributes? Thanks!